pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-sytem]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
@billdirks 
Question about the build-system settings.

https://github.com/great-expectations/great_expectations/pull/5506#discussion_r921784934